plugin-server/src/utils/event.ts

```diff
@@ -16,15 +15,11 @@ interface RawElement extends Element {
 }
 
 const convertDatabaseElementsToRawElements = (elements: RawElement[]): RawElement[] => {
-    for (const element of elements) {
-        if (element.attributes && element.attributes.attr__class) {
-            element.attr_class = element.attributes.attr__class
-        }
-        if (element.text) {
-            element.$el_text = element.text
-        }
-    }
-    return elements
+    return elements.map((element) => ({
```

Comment:
This used to mutate the object. No idea if this would have broken anything but it likely wasn't intentional, hence swapping to creating new objects

Reply:
see my comment in plugin-server/src/main/ingestion-queues/batch-processing/each-batch-webhooks.ts

Reply:
Ah I think I understood now. Pushed something to that effect